pkgs/by-name/yu/yuki-iptv/package.nix

```diff
@@ -0,0 +1,79 @@
+{
+  bash,
+  fetchurl,
+  ffmpeg,
+  gettext,
+  glib,
+  gobject-introspection,
+  lib,
+  mpv-unwrapped,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yuki-iptv";
+  version = "0.0.15";
+
+  src = fetchurl {
+    url = "https://codeberg.org/liya/yuki-iptv/archive/${version}.tar.gz";
+    sha256 = "sha256-8u8KoabExVqyj8ohGUl28SqIJYGfNLNl/tyHM9gwX88=";
+  };
+
+  buildInputs = [
+    ffmpeg
+    mpv-unwrapped
+  ];
+
+  nativeBuildInputs = [
+    gettext
+    python3.pkgs.wrapPython
+  ];
+
+  pythonPath = with python3.pkgs; [
```

Comment:
You are reinventing `python3Packages.buildPythonApplication` here. Try switching `stdenv.mkDerivation` to it (with `pyproject = false;`), change this to `dependencies`, and remove python workarounds below,

Reply:
@Aleksanaa can you offer guidance on what I have currently?

The main binary is a bash script that calls python. I can't seem to have it find it's python deps as is.